test/KernelInfo.rb

```diff
@@ -0,0 +1,67 @@
+# !/usr/bin/env ruby -w
+
+require 'rmagick'
+require 'test/unit'
+require 'test/unit/ui/console/testrunner' unless RUBY_VERSION[/^1\.9|^2/]
+
+class KernelInfoUT < Test::Unit::TestCase
+  setup do
+    @kernel = Magick::KernelInfo.new('Octagon')
+  end
+
+  def test_new
+    Magick::KernelInfoType.values do |kernel|
+      k = kernel.to_s.sub('Kernel', '')
+
+      if kernel == Magick::UserDefinedKernel
+        assert_raise(RuntimeError) { Magick::KernelInfo.new(k) }
+      else
+        assert_instance_of(Magick::KernelInfo, Magick::KernelInfo.new(k))
+      end
+    end
+    assert_raise(RuntimeError) { Magick::KernelInfo.new('') }
+    assert_raise(TypeError) { Magick::KernelInfo.new(42) }
+  end
+
+  def test_zero_nans
+    assert_nil(@kernel.zero_nans)
+  end
+
+  def test_unity_add
+    assert_nil(@kernel.unity_add(1.0))
+    assert_nil(@kernel.unity_add(12))
+    assert_raise(TypeError) { @kernel.unity_add('x') }
+  end
+
+  def test_show
+    assert_nil(@kernel.show)
+  end
+
+  def test_scale
```

Comment:
Seems this test causes SEGV with Ruby 2.3 on Windows

Reply:
I wonder if this is actually a bug in ruby 2.3 on Windows that this code is hitting. Could you get a backtrace with a debugger so we know where the segfault is raised?

Reply:
I want to know how to debug on Windows...